homeassistant/components/light/mqtt.py

```diff
@@ -85,7 +89,9 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
             CONF_STATE: config.get(CONF_STATE_VALUE_TEMPLATE),
             CONF_BRIGHTNESS: config.get(CONF_BRIGHTNESS_VALUE_TEMPLATE),
             CONF_RGB: config.get(CONF_RGB_VALUE_TEMPLATE),
-            CONF_COLOR_TEMP: config.get(CONF_COLOR_TEMP_VALUE_TEMPLATE)
+            CONF_COLOR_TEMP: config.get(CONF_COLOR_TEMP_VALUE_TEMPLATE),
+            CONF_COLOR_TEMP+"_COMMAND":
```

Comment:
The dictionary key will be `color_temp_command` to go alongside the existing dictionary key of `color_temp`. 

Honestly, I wasn't sure how best to handle this as it seemed more far-reaching than desired to make a change in the mqtt light component that required an addition to the homeassistant.const module. If that is the standard practice, then `CONF_COLOR_TEMP_COMMAND` could be added to `homeassistant/const.py`...